### ISSUE TITLE
General change in batch size Performance Question

### ISSUE THREAD

**Issue by Mazecreator** [tensorflow/tensorflow#15932]
I have opened a question on StackOverflow.  I suspect only a developer can answer the question.  The link to the question is here:
https://stackoverflow.com/questions/48099754/change-is-batch-size-impact-performance

The question is around Reinforcement Learning and if there will be a speed improvement if the batch size of the graph is not changed.  RL usually has a training batch and a "next_action" type of single sample (batch size =1).  The network is intermittently hit with either the training batch size or batch size of 1.  Does TF take a performance hit every time the graph sees a different batch size through the feed_dict()?  If this is the case, should 2 graphs be created so the batch size doesn't change from call to call?

If you post the answer here I will copy it to StackOverflow.

Thanks

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by Mazecreator** [comment]
@tensorflowbutler Updated

**Comment by tensorflowbutler** [comment]
The original poster has replied to this issue after the stat:awaiting response label was applied.

**Comment by drpngx** [comment]
@ebrevdo wanna take that one?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @ebrevdo: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @ebrevdo: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @ebrevdo: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by Mazecreator** [comment]
Any thoughts on this or should we close it as unknown?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @ebrevdo: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
